config/puma.rb

```diff
@@ -7,9 +7,14 @@
 threads_count = ENV.fetch("RAILS_MAX_THREADS") { 5 }
 threads threads_count, threads_count
 
+# Specifies the `worker_timeout` threshold that Puma will use to wait before
+# terminating a worker in development environments.
+#
+worker_timeout 3600 if ENV.fetch("RAILS_ENV", "development") == "development"
```

Comment:
1 hour?

Reply:
This is the default in Rails 6.1 
https://github.com/rails/rails/blob/v6.1.7.7/railties/lib/rails/generators/rails/app/templates/config/puma.rb.tt#L11-L14

see https://github.com/rails/rails/pull/40503

Reply:
maybe good for debugging